tests/Microsoft.Bot.Builder.Azure.Tests/AzureBlobTranscriptStoreTests.cs

```diff
@@ -2,116 +2,502 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Reflection;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Auth;
-using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Auth;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Bot.Schema;
+using Moq;
+using Newtonsoft.Json;
 using Xunit;
-using Xunit.Abstractions;
-using Xunit.Sdk;
 
-// These tests require Azure Storage Emulator v5.7
-// The emulator must be installed at this path C:\Program Files (x86)\Microsoft SDKs\Azure\Storage Emulator\AzureStorageEmulator.exe
-// More info: https://docs.microsoft.com/azure/storage/common/storage-use-emulator
 namespace Microsoft.Bot.Builder.Azure.Tests
 {
-    [Trait("TestCategory", "Storage")]
-    [Trait("TestCategory", "Storage - BlobTranscripts")]
-    public class AzureBlobTranscriptStoreTests : TranscriptStoreBaseTests, IAsyncLifetime
+    public class AzureBlobTranscriptStoreTests
```

Comment:
We should probably keep the _TestCategories_ from the old tests. I didn't notice this before, sorry.